Lib/test/test_genericalias.py

```diff
@@ -2,12 +2,14 @@
 
 import unittest
 from collections import defaultdict, deque
+from io import IOBase
+from re import Pattern, Match
 
 class BaseTest(unittest.TestCase):
     """Test basics."""
 
     def test_subscriptable(self):
-        for t in tuple, list, dict, set, frozenset, defaultdict, deque:
+        for t in tuple, list, dict, set, frozenset, defaultdict, deque, IOBase, Pattern, Match:
```

Comment:
TODO: I suppose IOBase subclasses BufferedIOBase, RawIOBase and TextIOBase should not support indexing.

Reply:
There is also Python implementation `_pyio.IOBase`. I am not sure what to do with it. Maybe just ignore for now.

Reply:
I added `__class_getitem__` to `_pyio.IOBase` in 35cd5f477c9da985880130d72297335e010450e4.

I'm not sure whether it matters much that we take the `__class_getitem__` away from the various subclasses (let the type checker worry about that).